dev-docs/modules/userid-submodules/utiqMtp.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: userid
+title: UtiqMtp ID
+description: UtiqMtp User ID sub-module
+useridmodule: utiqMtpIdSystem
+---
+
+Utiq generates unique tokens, enabling improved efficiency in programmatic advertising while safeguarding transparency and control for end customers via `consenthub.utiq.com`. A website visitor’s Utiq ID is generated based on network identifiers provided by network operators and requires explicit user consent.
+
+Utiq provides different tokens type, known as 'adtechpass' and 'martechpass'. UtiqMtpId module helps publishers support and pass the 'martechpass' token to their SSP partners of choice. 'martechpass' is a first party ID meant to be enable first party audience in the SSP and shouldn't be forwarded to other parties.
+
+Both Utiq and UtiqMtpId user id submodules can be used in parallel as both supports different use cases and scenarios.
+
+Utiq is also the brand name of the service, which is provided by Utiq SA/NV.
+
+## UtiqMtp ID configuration
```

Comment:
as mentioned in the pbjs repo, it appears that additional installation steps are required by publishers beyond just the prebid config. as such, this should be mentioned here so publishers don't think this will "just work" by turning it on.

Reply:
just added some extra information in the documentation to inform publishers about necessity to instal some external script as well as link to utiq contact form if they are interested.